src/web/app/components/question-types/question-edit-answer-form/msq-question-edit-answer-form.component.ts

```diff
@@ -52,30 +59,27 @@ export class MsqQuestionEditAnswerFormComponent
   updateSelectedAnswers(index: number): void {
     this.isMsqOptionSelected[index] = !this.isMsqOptionSelected[index];
```

Comment:
Because of `ngOnChanges`, you don't need to update the array manually. That is to say; updating the answers will automatically update the `isMsqOptionSelected`.